tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -111,7 +111,7 @@ static void test_all_instances(counter_test_func_t func,
 		if ((capability_check == NULL) ||
 		     capability_check(devices[i])) {
 			TC_PRINT("Testing %s\n", devices[i]);
-			func(devices[i]);
+			/* func(devices[i]); */
```

Comment:
that looks suspicious, as if test functions were not executed. Did you change the mechanism of executing tests?

Reply:
Good catch! I commented it out for some testing locally. I must have accidentally have committed it. Fixed.

I need to rerun all my tests to ensure none of them were falsely run with this change in.

Reply:
which boards are you using? Any nordic boards?

Reply:
I am using the nrf52840_pca10056. I will retest when I get back to the lab.

Reply:
@nordic-krch Current implementation retested and verified on nrf52840_pca10056. Please re-review.